app/views/settings/ext_cert/index.html.slim

```diff
@@ -0,0 +1,138 @@
+= render 'settings/apply'
+
+h2 External Certificates
+
+p Enable the use of External SSL Certificates for CaaSP and Kubernetes services.  Certificates and matching keys must be in armored PEM format.
+.alert.alert-warning role='alert'
+ p Remember to add a Root CA Certificate to the System Wide Certificates corresponding to any new External Certificates
+
+= form_for :external_certificate, url: settings_ext_cert_index_path, method: :post do |f|
+  br
+  .panel.panel-default
+    .panel-heading
+      h3.panel-title Velum SSL Certificate and Key
+    .panel-body
+      - if @velum_cert
+        .cert-table
+          h5 Current Velum SSL Certificate Info:
+          table
+            - @velum_cert.each do |k, v|
```

Comment:
I wouldn't show `Error Certificate not available, please upload a certificate`. Maybe showing only `Certificate not available, please upload a certificate`. `Error` is definitely not a good word to be used.

Reply:
I would probably use a simple `p` instead of the `table`.

Reply:
I made it `Notice` instead.

Reply:
![screenshot-20190109143716-523x85](https://user-images.githubusercontent.com/188554/50918019-d57f2f00-141d-11e9-9f24-e02a98c5c78b.png)

This is the problem that I've mentioned that was going to happen if we don't differentiate the way to display the content.

Reply:
I'm currently using the same mechanism to inject an error message [here](https://github.com/danielorf/velum/blob/external_certs_front_end/app/controllers/settings/ext_cert_controller.rb#L163).  Do we need to make sure this unlikely state has the ideal spacing?  If so, I will have to add logic to the slim code looking for each of these states - and I'm hesitant to spread the logic across multiple files if I can keep it in the controller.

Reply:
Unfortunately the logic must go somewhere in this case unless you see a way to differentiate them without adding that. We could differentiate using a css class but to define the css class we need some logic. So it's inevitable. You can use helpers to do that instead of doing it in the template or controller. That's the concept behind rails helpers.

Reply:
I fixed this at the expense of making the key section a table:
![image](https://user-images.githubusercontent.com/10256394/50940646-9a3d2a00-1436-11e9-84a2-378d9430de11.png)

Reply:
The key info was fine for me, the problem was with the case of `Certificate not available, please upload a certificate` that you've fixed that and looks great.

Do we display any other info besides "valid key" for the keys to treat this data as a table? Couldn't we simply display "Your key is valid"? Why is it important to have it displayed like that? If there's other info to be displayed I'll understand to have it as tabulated data, otherwise revert it and kill the spacing for key as you did for the not uploaded file.